commit d36d642c20f28bb22cae416c12d19b5231f9c050 (refs/changes/45/5245/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-12-26T23:46:14+00:00 (10 months ago)
    
    TRITON-1044 cns should use newer redis server and client, paginate AXFRs
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Isaac Davis <isaac.davis@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index 8e33820..2e41d6d 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -181,9 +181,9 @@ cat <<EOF
     "image_name": "cns",
     "image_description": "Triton Container Naming Service",
     "image_version": "1.0.0",
-    "image_uuid": "b4bdc598-8939-11e3-bea4-8341f6861379",
+    "image_uuid": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
     "pkgsrc": [
-      "redis-2.6.16"
+      "redis-3.0.5"
     ],
     "repos": [
         {"url": "$MG_GIT_PREFIX/triton-cns.git", "dir": "triton-cns"}
